plugins-compat-tester/src/test/java/org/jenkins/tools/test/ScmConnectionSpecialCasesTest.java

```diff
@@ -109,30 +109,6 @@ public void shouldGithubDotComBeFollowedBySlashes() throws Throwable{
     }
 
     @Test
-    public void shouldGithubBeAccessedWithGitProtocol() throws Throwable{
```

Comment:
Since we are not already applying the transformation, these tests do not make sense.